From 3c3d69e71de94ab8f3efbc8b80bd307f86b81319 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Tue, 25 Aug 2015 15:23:44 +0900
Subject: [PATCH 04/85] arm64: dts: salvatorx: Add EXTAL clock override

Override the "extal_clk" clock with the frequency and accuracy of the
crystal oscillator on the Salvator-X board.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts
index 1b00390..2634667 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvatorx.dts
@@ -31,6 +31,10 @@
 	};
 };
 
+&extal_clk {
+	clock-frequency = <16666666>;
+};
+
 &scif2 {
 	status = "okay";
 };
-- 
1.7.9.5

